modules/oe_paragraphs_illustrations_lists/config/install/paragraphs.paragraphs_type.oe_illustration_item_flag.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
+status: true
+dependencies: {  }
+id: oe_illustration_item_flag
+label: 'Illustration item with flag'
+icon_uuid: null
+icon_default: null
+description: "Single illustration item with flag, to be used exclusively in conjunction with the \"Illustration list with flags\" paragraph type."
```

Comment:
```suggestion
description: 'Single illustration item with flag, to be used exclusively in conjunction with the "Illustration list with flags" paragraph type.'
```

Reply:
It was generated automatically. Also /admin/config/development/configuration/single/export generates this code. So I think, it's better to keep it as is.